CHANGES.rst

```diff
@@ -96,6 +96,20 @@ extract_1d
   precedence is given for command line override, reference file settings, and
   internal decisions of the appropriate setting (in that order). [#7466]
 
+jump
```

Comment:
This needs to go up at the top of the log, in the 1.10.1 (unreleased section), since 1.10.0 is already out the door.